pyproject.toml

```diff
@@ -223,4 +232,8 @@ filterwarnings = [
     "error:::zarr.*",
     "ignore:PY_SSIZE_T_CLEAN will be required.*:DeprecationWarning",
     "ignore:The loop argument is deprecated since Python 3.8.*:DeprecationWarning",
+    "ignore:Creating a zarr.buffer.gpu.*:UserWarning",
+]
+markers = [
+    "gpu: mark a test as requiring CuPy and GPU"
```

Comment:
Make it is worth using `cuda` here?

Reply:
You're thinking, @jakirkham, that then there could be a multiplicity of these.